releasenotes/notes/horizon-extensions-in-tree-91f2ed817fde11aa.yaml

```diff
@@ -2,3 +2,8 @@
 other:
   - The horizon extensions are now provided in the
     rpc-openstack repository.
+upgrade:
```

Comment:
@git-harry can confirm or deny, but I don't think we're supposed to amend release notes after the fact. Rather we should be generating new ones.